pkg/apis/configuration/validation/virtualserver_test.go

```diff
@@ -69,215 +69,6 @@ func TestValidateVirtualServer(t *testing.T) {
 	}
 }
 
```

Comment:
any specific reason to remove these unit tests other than datatype change?

Reply:
The tests are removed because of datatype change (they are not relevant now). Now we have boolean and it can be either `true` or `false`. Validation of this type is enforced  in the OpenAPI spec (updated CRD for VirtualServer).